CMakeLists.txt

```diff
@@ -377,39 +382,6 @@ add_custom_target(cudaq-headers)
 set_target_properties(cudaq-headers PROPERTIES FOLDER "Misc")
 add_custom_target(cudaq-doc)
 
-# Install
-# ==============================================================================
-
-# If building normally, this will add the required 
-# system headers to the cudaq-quake include paths.
-if (NOT CUDAQ_BUILD_RELOCATABLE_PACKAGE)
```

Comment:
Will removing this break the tools when they don't use these scripts and dockerfiles?